common/src/main/java/com/mx/common/connect/Response.java

```diff
@@ -153,13 +155,18 @@ public final void setHeaders(MultiValueMappable<String, String> multiValueMappab
     this.headers = new MultiValueMap<>(multiValueMappable);
   }
 
+  @SuppressWarnings("unchecked")
   public final RESP throwException() {
     if (Objects.nonNull(exception)) {
       if (exception instanceof MdxApiException) {
         throw (MdxApiException) exception;
       }
 
-      throw new MdxApiException("Unprocessed request exception", HttpStatus.INTERNAL_SERVER_ERROR, true, exception);
+      if (PathRequestException.class.isAssignableFrom(exception.getClass())) {
```

Comment:
Minor nitpick. I think `if (exception instanceof PathRequestException)` is easier to read, and it's consistent with the code directly above it.